tests/widgets_gauge.rs

```diff
@@ -28,6 +28,46 @@ fn widgets_gauge_renders() {
             f.render_widget(gauge, chunks[1]);
         })
         .unwrap();
+    let expected = Buffer::with_lines(vec![
+        "                                        ",
+        "                                        ",
+        "  ┌Percentage────────────────────────┐  ",
+        "  │              ▋43%                │  ",
```

Comment:
Nice, could we also check the color behavior ? Especially when we are at 50% and the label may have different colors.

Reply:
I'm not sure how to get the color from the terminal/backend for an assert. Is there an example of this somewhere else?

Reply:
You can simply modify the cell of the expected `Buffer`. See [this](https://github.com/fdehau/tui-rs/blob/5a9b59866bbe93dad00309e4ffc449a21601972d/tests/widgets_list.rs#L33) for an example.